demos/TransferLearning/TransferLearningDemo.py

```diff
@@ -263,143 +151,46 @@ def init_images(self):
-                f, ax = plt.subplots(3, 3, figsize=(15, 15))
-
-                for i in range(0, 9):
-                    test_image = os.path.join(self.test_dir,
-                                              self.test_generator.filenames[self.wrong_guesses[i]])
-                    imgRGB = mpimg.imread(test_image)
-
-                    predicted_class = "Dog" if self.predicted_class_indices_init[
-                        self.wrong_guesses[i]] else "Cat"
-
-                    ax[i // 3, i % 3].imshow(imgRGB)
-                    ax[i // 3, i % 3].axis('off')
-                    ax[i // 3, i % 3].set_title("Predicted:{}".format(predicted_class), color='r')
-
-                    if predicted_class.lower() in self.test_generator.filenames[
-                            self.wrong_guesses[i]]:
-                        ax[i // 3, i % 3].set_title("Predicted:{}".format(predicted_class),
-                                                    color='g')
-                display(plt)
-
-    def on_model_change(self, change):
-        if change['type'] == 'change' and change['name'] == 'value':
-            self.classify_button.disabled = True
-
-            if change['new'] == 'ResNet50':
-                self.epoch_slider.min = 1
-                self.epoch_slider.max = 16
-                self.epoch_slider.value = 2
-            if change['new'] == 'MobileNet v2':
-                self.epoch_slider.min = 2
-                self.epoch_slider.max = 50
-                self.epoch_slider.value = 10
-
-    def on_ngraph_change(self, change):
-        if change['type'] == 'change' and change['name'] == 'value':
-            i = self.ngraph_backends.index(change['new'])
-
-            if self.ngraph_backends[i] == 'DISABLED':
-                self.use_ngraph = False
-                ngraph_bridge.disable()
-            else:
-                self.use_ngraph = True
-                ngraph_bridge.enable()
-                ngraph_bridge.set_backend(self.ngraph_backends[i])
-
-    def init_gui(self):
-        self.model_dropdown.observe(self.on_model_change)
-        self.ngraph_dropdown.observe(self.on_ngraph_change)
-        self.train_button.on_click(self.train_model)
-        self.shuffle_button.on_click(self.init_model)
-        self.classify_button.on_click(self.classify)
-        self.fine_tune_button.on_click(self.train_model)
-
-        return self.gui
-
-    def train_model(self, b=None, epochs=1, batch_size=32, model='ResNet50', fine_tune_at=0):
-        if self.gui:
-            self.train_button.disabled = True
-            self.epochs = self.epoch_slider.value
-            self.batch_size = self.batch_slider.value
-            model = self.model_dropdown.value
-            self.progress_text.value = ''
-        else:
-            self.epochs = epochs
-            self.batch_size = batch_size
+    def setup_ngraph_bridge(self, backend):
+        # Enviornment variables
+        os.environ['PLAIDML_USE_STRIPE'] = '1'
```

Comment:
This probably won't have any effect unless it happens before `import ngraph_bridge` occurs.